src/lib.rs

```diff
@@ -9,6 +9,7 @@ extern crate flate2;
 extern crate failure;
 extern crate itertools;
 extern crate rayon;
+extern crate regex;
```

Comment:
Are you using this?

Reply:
no - must have meant to add this to a different project.